guides/personal_mixed_skin.md

```diff
@@ -0,0 +1,56 @@
+
+Apart from these differences above, the process is the same as described in the `Mixing images` section of this guide.
+
+## Mixing sounds
+
+The steps for mixing sounds are also largely similar to the `Mixing images` section of this guide, with the difference that the files you are replacing will be .mp3 or .wav instead of .png. Follow the instructions in the `Mixing images` section, using sound files instead.
+
+## The skin.ini
+
+The skin.ini is something that you will have to check when you are mixing a skin to make sure it looks right when selected. There are a few things that you should check:
+
+- `AnimationFramerate:` - This dictates how many frames of an animation will be displayed in one second. If the animation(s) you are copying over to your mixed skin relies on this command being set to 30 frames per second, 60 frames per second, etc., you will have to adjust other animations in your skin to compensate for this. For example, if we have to use an `AnimationFramerate` setting of `60` but our `followpoint-x.png` only goes up to 9 (10 frames), you will need to have six copies of each frame. Copy `followpoint-0` five times and name these copies `followpoint-1`, `followpoint-2`, and so on and so forth. Then do the same for the other 9 frames. Your followpoints will now look normal as they animate at the same framerate as your `AnimationFramerate` setting.
+- `Combo[x]:` - This dictates your combo colors, i.e., the tint that is applied to your `hitcircle.png`, and `sliderstartcircle.png` in gameplay. This setting goes up to 8 and takes `r, g, b` color codes. Note that the combo colors start at `Combo2` and loop back around to finish on `Combo1`. The color is applied multiplicatively, so if you want your `hitcircle.png`, and `sliderstartcircle.png` to be un-tinted, use `Combo1: 255, 255, 255`.
+- `HitCirclePrefix:`, `ScorePrefix:`, `ComboPrefix:` - This dictates the prefix for the default, score, and combo images. Make sure to set these to the same prefix as the skin you copied the files from, or they will not display in game. Note that if your default, score, and/or combo images are in a nested folder, you will have to specify this here. For example, if your `default-x.png` images are in a folder called `num`, and are prefixed with `c-`, set `HitCirclePrefix:` to `num\c`.
+- `HitCircleOverlap:`, `ScoreOverlap:`, `ComboOverlap:` - This dictates how many pixels the default, score, and combo images will overlap with each other. Higher values will lead to a greater overlap (to a certain point) and lower values will lead to a lesser overlap. Note that this value is for the SD overlap, it will be doubled for HD resolutions and images. Set this to the same value as in the skin you mixed default, score, and combo images from.

```

Comment:
wdym with to a certain point

they also dont have to set it to the same, maybe they wanna tweak the distance, would change to "Set this to the same value as in the skin you take default, score, and combo images from if you went them to look the same." or something similar